index.css

```diff
@@ -0,0 +1,53 @@
+/* Importing All Components CSS */
+@import "/components.css";
+
+/* Keyframes for Load Animation */
+
+@keyframes slideInFromBottom {
+	0% {
+		transform: translateY(2rem);
+		opacity: 0;
+	}
+	100% {
+		transform: translateY(0);
+		opacity: 1;
+	}
+}
+
+/* Index Container */
+.index-container {
+	display: flex;
+	flex-direction: column;
+	width: 100vw;
+	height: 100vh;
+}
+
+/* Index Main */
+.index-main {
+	display: flex;
+	height: 100vh;
+	margin: var(--space-lg);
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	gap: var(--space-sm);
```

Comment:
A small suggestion here would be that as there are many properties in a single class you can sort them alphabetically as that can help in debugging as well as code maintenance.

Reply:
done. thank you.